probes/releasesHaveProvenance/def.yml

```diff
@@ -0,0 +1,28 @@
+# Copyright 2023 OpenSSF Scorecard Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+id: releasesHaveProvenance
+short: Check that the projects releases have provenance.
+motivation: >
+  Provenance give users security-critical, verifiable information so that consumers can verify their artifacts before consuming them.
+implementation: >
+ The probe checks for a provenance file in the assets of the last five releases.
+outcome:
+  - For each of the last 5 releases, the probe returns OutcomePositive, if the release has a provenance file in the release assets.
+  - For each of the last 5 releases, the probe returns OutcomeNegative, if the the release does not have a provenance file in the release assets.
+  - If the project has no releases, the probe returns OutcomeNotApplicable.
+remediation:
+  effort: Medium
+  text:
+    - Generate provenance and add it alongside the release assets.
```

Comment:
we need to explain how to do that step-by-step

Reply:
Not really step-by-step but added in https://github.com/ossf/scorecard/pull/3610/commits/f6fdd13d0a56274395529d3385a0152465920d56. Too plain?